RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ImportMnemonicControllingAccounts.swift

```diff
@@ -23,8 +23,29 @@ public struct ImportMnemonicControllingAccounts: Sendable, FeatureReducer {
 		) {
 			self.isMainBDFS = isMainBDFS
 			self.entitiesControlledByFactorSource = entitiesControlledByFactorSource
+
+			let accounts: IdentifiedArrayOf<Profile.Network.Account> = if isMainBDFS, let babylonAccounts = entitiesControlledByFactorSource.babylonAccounts {
+				babylonAccounts.rawValue
+			} else if isMainBDFS, entitiesControlledByFactorSource.babylonAccounts == nil, let olympiaAccounts = entitiesControlledByFactorSource.olympiaAccounts {
+				// Fun MEGA edge case:
```

Comment:
TODO: Check with Matt if we want this: https://rdxworks.slack.com/archives/C031A0V1A1W/p1702028132533319?thread_ts=1701954053.387289&cid=C031A0V1A1W